ccm_web/client/src/components/FeatureCard.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
+import { makeStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+// import CardActions from '@material-ui/core/CardActions'
```

Comment:
I hear ya, seems like this is  inevitably going to be uncommented when it's made clickable so I just left it.  Maybe it bothers you more that I left it in than it would bother me to delete it and add it back in lol